sharepoint/sharepoint-ps/sharepoint-online/Get-SPOContainer.md

```diff
@@ -50,7 +50,7 @@ Get-SPOContainer [[-Identity] <ContainerSiteURL>]
 
 The `Get-SPOContainer` cmdlet retrieves and returns a list of Containers and details of an individual Container created under a SharePoint Embedded application. This command is available only in SharePoint Online Management Shell version 16.0.24211.12000 or higher to run this cmdlet.
 
-You must be a SharePoint Online Administrator to run this cmdlet.
+You must be a Global reader/Global Administrator/SharePoint Embedded Administrator to run this cmdlet.
```

Comment:
>Global reader/Global Administrator/SharePoint Embedded Administrator [](http://example.com/codeflow?start=14&length=68)

"Global Reader or SharePoint Embedded Administrator"

do not include Global Administrator -- it's being de-emphasized across all the documentation